tests/js/spec/views/eventsV2/miniGraph.spec.tsx

```diff
@@ -59,6 +59,14 @@ describe('EventsV2 > MiniGraph', function () {
     });
     // @ts-expect-error
     eventView = EventView.fromSavedQueryOrLocation(undefined, location);
+
+    // @ts-expect-error
+    MockApiClient.clearMockResponses();
```

Comment:
I was having errors on the other tests complaining that the endpoint wasn't mocked. I suppose adding the mocking to trigger the error makes this cleanup necessary?

Reply:
👍  looks good to me